hawc/apps/riskofbias/serializers.py

```diff
@@ -173,13 +172,9 @@ class Meta:
         fields = ("id", "riskofbias", "metric")
 
     def create(self, validated_data):
-        override = models.RiskOfBiasScore.objects.create(
-            riskofbias=validated_data["riskofbias"],
-            metric=validated_data["metric"],
-            is_default=False,
-            label="",
-        )
-        return override
+        score = validated_data["metric"].build_score(validated_data["riskofbias"], True)
```

Comment:
@rabstejnek I used our `build_score` method here too just so scores are all made form the same place; let me know if that looks ok too

Reply:
Looks good, but since this is the override creator we should make `is_default` false.